src/Sylius/Bundle/ImportExportBundle/Behat/ImportExportContext.php

```diff
@@ -0,0 +1,637 @@
+            false === strpos($headers, 'username') ||
+            false === strpos($headers, 'roles')
+        ) {
+            throw new \Exception('The file is not valid');
+        }
+    }
+
+    /**
+     * @Then I should have :numberOfUsers users in a database
+     */
+    public function iHaveMoreUsersInADatabase($numberOfUsers)
+    {
+        $repository = $this->getRepository('user');
+
+        $numberOfUsersInDatabase = count($repository->findAll());
+        if ((int) $numberOfUsers !== $numberOfUsersInDatabase) {
+            $message = sprintf(
+                '%s users was expected to be found, but %s was found in a database',
+                $numberOfUsers,
+                $numberOfUsersInDatabase
+            );
+
+            throw new \Exception($message);
+        }
+    }
+
+    /**
+     * @Then Export job for export profile :exportProfile should be created
+     */
+    public function exportJobForThisExportProfileShouldBeCreated($exportProfileName)
+    {
+        $this->profile = $this->getRepository('export_profile')->findOneBy(array('name' => $exportProfileName));
+        $this->filePath = $this->profile->getWriterConfiguration()['file'];
+        $this->job = $this->profile->getJobs()->current();
+        $this->type = 'export';
+
+        if (false === $this->job) {
+            throw new \Exception('Export job has not been created');
+        }
+    }
+
+    /**
+     * @Then I should be on its details page
+     */
+    public function iShouldBeOnItsDetailsPage()
+    {
+        if (null === $this->job || null === $this->profile) {
+            throw new \Exception('Job or profile was not set. Make "Export job for export profile " " should be created" before you use this step.');
+        }
+
+        $path = sprintf('sylius_backend_%s_job_show', $this->type);
+
+        $this->getSession()->visit($this->generatePageUrl($path,
+            array(
+                'profileId' => $this->profile->getId(),
+                'id' => $this->job->getId(),
+            )
+        ));
+    }
+
+    /**
+     * @Then Import job for import profile :importProfileName should be created
+     */
+    public function importJobForImportProfileShouldBeCreated($importProfileName)
+    {
+        $this->profile = $this->getRepository('import_profile')->findOneBy(array('name' => $importProfileName));
+        $this->filePath = $this->profile->getReaderConfiguration()['file'];
+        $this->job = $this->profile->getJobs()->current();
+        $this->type = 'import';
+
+        if (false === $this->job) {
+            throw new \Exception('Import job has not been created');
+        }
+    }
+
+    /**
+     * @param string    $name
+     * @param string    $description
+     * @param string    $code
+     * @param string    $reader
+     * @param string    $readerConfiguration
+     * @param string    $writer
+     * @param string    $writerConfiguration
+     * @param bool|true $flush
```

Comment:
Wrong typehint, `bool` or `boolean` is enough. If you fix it, there will be identation issue too :)